src/components/events-carrosel/index.js

```diff
@@ -48,44 +44,49 @@ const Events = () => {
         breakpoints={{
           '@0.00': {
             slidesPerView: 1,
-            spaceBetween: 1
+            spaceBetween: 1,
           },
           '@0.75': {
             slidesPerView: 2,
-            spaceBetween: 2
+            spaceBetween: 2,
           },
           '@1.00': {
             slidesPerView: 3,
-            spaceBetween: 3
-          }
+            spaceBetween: 3,
+          },
         }}
         modules={[Navigation]}
         className="mySwiper"
       >
         <section>
           <div className="swiper-slide">
             <ul>
-              {attributesEvents.map((events, key) =>
+              {attributesEvents.map((events, key) => (
```

Comment:
Estamos tratando de um único evento no escopo desta função:

```suggestion
              {attributesEvents.map((event, key) => (
```

Reply:
Alteração feita.